.github/workflows/clp-core-build.yaml

```
@@ -163,16 +163,21 @@ jobs:
         run: "chown $(id -u):$(id -g) -R ."
         shell: "bash"
 
-      - uses: "./.github/actions/clp-core-build"
+      - uses: "./.github/actions/clp-core-image-run"
         env:
           OS_NAME: "centos-stream-9"
         with:
           image_name: "${{env.DEPS_IMAGE_NAME_PREFIX}}${{env.OS_NAME}}"
-          use_shared_libs: "${{matrix.use_shared_libs}}"
           use_published_image: >-
             ${{needs.filter-relevant-changes.outputs.centos_stream_9_image_changed == 'false'
             || (github.event_name != 'pull_request' && github.ref == 'refs/heads/main')}}
-          upload_binaries: "false"
+          run_command: >-
+            task deps:core &&
+            python3 /mnt/clp/components/core/tools/scripts/utils/build-and-run-unit-tests.py
```

```suggestion
            task deps:core
            && python3 /mnt/clp/components/core/tools/scripts/utils/build-and-run-unit-tests.py
```

Optional, but matches our practices of putting the operator at the start of the line in a multiline expression.